<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIAuxiliaryWnd</class>
 <widget class="QWidget" name="UIAuxiliaryWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIAuxiliaryWnd</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="title_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/LoginWindow/images/logo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pic_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pic_btn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/LoginWindow/images/t_photo.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="min_pushButton">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/LoginWindow/images/login_min.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_pushButton">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/LoginWindow/images/login_close.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>11111</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/LoginWindow/images/alpha.png);</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="today_radioButton">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>今日安排</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton:checked {border-image: url(:/LoginWindow/images/tabToday_hover.png);}

QRadioButton:unchecked {border-image: url(:/LoginWindow/images/tabToday_nor.png);}

QRadioButton:unchecked:hover {border-image: url(:/LoginWindow/images/tabToday_hover.png);}

QRadioButton::indicator:checked {border-image: url(:/LoginWindow/images/.png);}

QRadioButton::indicator:unchecked {border-image: url(:/LoginWindow/images/.png);}

QRadioButton::indicator:unchecked:hover {border-image: url(:/LoginWindow/images/.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="all_radioButton">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>直播课</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton:checked {border-image: url(:/LoginWindow/images/tabAll_hover.png);}

QRadioButton:unchecked {border-image: url(:/LoginWindow/images/tabAll_nor.png);}

QRadioButton:unchecked:hover {border-image: url(:/LoginWindow/images/tabAll_hover.png);}

QRadioButton::indicator:checked {border-image: url(:/LoginWindow/images/.png);}

QRadioButton::indicator:unchecked {border-image: url(:/LoginWindow/images/.png);}

QRadioButton::indicator:unchecked:hover {border-image: url(:/LoginWindow/images/.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="oneToOne_radioButton">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>23</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>一对一</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton:checked {border-image: url(:/LoginWindow/images/tab1v1_hover.png);}

QRadioButton:unchecked {border-image: url(:/LoginWindow/images/tab1v1_nor.png);}

QRadioButton:unchecked:hover {border-image: url(:/LoginWindow/images/tab1v1_hover.png);}

QRadioButton::indicator:checked {border-image: url(:/LoginWindow/images/.png);}

QRadioButton::indicator:unchecked {border-image: url(:/LoginWindow/images/.png);}

QRadioButton::indicator:unchecked:hover {border-image: url(:/LoginWindow/images/.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="line_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(229, 229, 229);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="today_widget" native="true">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/LoginWindow/images/alpha.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="today_scrollArea">
           <property name="styleSheet">
            <string notr="true">QScrollArea{border: 1px solid #ffffff;background-color: rgb(255, 255, 255);}</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>72</width>
              <height>574</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="all_widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/LoginWindow/images/alpha.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">QScrollArea{border: 1px solid #ffffff;background-color: rgb(255, 255, 255);}
</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>73</width>
              <height>574</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="oneToOne_widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/LoginWindow/images/alpha.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="oneToOne_scrollArea">
           <property name="styleSheet">
            <string notr="true">QScrollArea{border: 1px solid #ffffff;background-color: rgb(255, 255, 255);}
</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>72</width>
              <height>574</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="loginwindow.qrc"/>
 </resources>
 <connections/>
</ui>
